docs/pim/symbol_attribute_type.md

```diff
@@ -0,0 +1,138 @@
+description: Create a symbol attribute type that enables for the efficient representation of string-based values while enforcing their format in product specifications.
+---
+
+# Symbol attribute type
+
+In product specifications, the symbol attribute type enables the efficient representation of string-based data and enforces their format.
+
+This feature allows you to store standard product identifiers (such as EAN or ISBN) in the [Product Information Management](pim_guide.md) system.
+
+## Installation
+
+### Download the bundle
+
+To get the most recent stable version of this bundle, open a command terminal, navigate to your project directory, and run the following command:
+
+``` bash
+composer require ibexa/product-catalog-symbol-attribute
+```
+
+### Enable the bundle
+
+Symfony Flex enables and configures the `IbexaProductCatalogSymbolAttributeBundle` automatically.
+If you don't use it, you can manually enable this bundle by adding the line below to the Kernel of your project:
+
+``` php
+// config/bundles.php
+
+return [
+   // ...
+   Ibexa\Bundle\ProductCatalogSymbolAttribute\IbexaProductCatalogSymbolAttributeBundle::class => ['all' => true],
+   // ...
+];
+```
+
+### Update database schema
+
+To store symbol attribute values, the `IbexaProductCatalogSymbolAttributeBundle` needs an extra table.
+The following SQL query can be used to build the required database structure:
+
+``` sql
+create table ibexa_product_specification_attribute_symbol (
+    id int not null primary key,
+    value varchar(255) null,
+    constraint ibexa_product_specification_attribute_symbol_fk
+        foreign key (id) references ibexa_product_specification_attribute (id)
+            on update cascade on delete cascade
+) collate = utf8mb4_unicode_520_ci;
+
+create index ibexa_product_specification_attribute_symbol_value_idx
+    on ibexa_product_specification_attribute_symbol (value);
+```
+
+### Create symbol attribute definition (optional)
+
+Now, you're able to define symbol attributes at this point.
+
+To create symbol attribute definition, in the back office, go to **Product catalog** -> **Attributes**, and click **Create**.
+Then, choose **Symbol** attribute type.
+
+## Build-in symbol attribute formats
+
+The built-in symbol attribute formats in `ibexa/product-catalog-symbol-attribute` are listed below:
+
+| Name | Description | Example |
+|-----------------|-----------------|-----------------|
+| Generic | Accepts any string value  | #FR1.2 |
+| Generic (alphabetic characters only) | Accepts any string value that contais only letters  | ABCD |
+| Generic (digits only) | Accepts any string value that contais only digits  | 123456 |
+| Generic (alphanumeric characters only) | Accepts any string value that contains only letters or digits | 2N6405G |
+| Generic (hexadecimal digits only) | Accepts any string value that contains only hexadecimal digits (digits or A-F characters) | DEADBEEF |
+| EAN-8 | European Article Number (8 characters)  | 29033706 |
```

Comment:
This EAN is invalid (see https://issues.ibexa.co/browse/IBX-10279 or https://eancheck.com/)

Let's use the one from https://en.wikipedia.org/wiki/EAN-8 : 96385074